pkg/apis/pipeline/v1beta1/resource_types.go

```diff
@@ -123,10 +123,10 @@ type PipelineResourceResult struct {
 	Key          string `json:"key"`
 	Value        string `json:"value"`
 	ResourceName string `json:"resourceName,omitempty"`
-	// This field should be deprecated and removed in the next API version.
+	// The field ResourceName should be deprecated and removed in the next API version.
```

Comment:
Is this right?  I have to confess I'm a bit confused by these fields and #2694 doesn't make it totally clear.  @dlorenc is it ResourceName or ResourceRef that's supposed to be deprecated?